kotsadm/web/src/config-ui/subNavConfig.js

```diff
@@ -90,5 +90,13 @@ export default [
     displayRule: (watch, isVeleroInstalled) => {
       return watch.allowSnapshots && isVeleroInstalled;
     }
+  },
+  {
+    tabName: "settings",
```

Comment:
ditto

Reply:
This one is "Settings" because ultimately other items will migrate over to this tab and it won't be just app identity settings.